wcivf/apps/feedback/forms.py

```diff
@@ -12,12 +12,15 @@ def __init__(self, *args, **kwargs):
 
     class Meta:
         model = Feedback
-        fields = ["found_useful", "sources", "comments", "source_url"]
+        fields = ["found_useful", "sources", "vote", "comments", "source_url"]
 
     found_useful = forms.ChoiceField(
         choices=FOUND_USEFUL_CHOICES,
         widget=forms.RadioSelect(attrs={"data-toggle": "button"}),
     )
-
+    vote = forms.ChoiceField(
```

Comment:
For the answers to populate through to the slack channel, the `batch_feedback_to_slack` management command will need to be updated to add in this new field....

... which has made me realise we haven't been getting any feedback lately because the new environment doesn't have the cron enabled! I will need to reenable that